.github/workflows/test.yaml

```diff
@@ -66,23 +102,63 @@ jobs:
           # `python_requires` from `setup.cfg`.
           - python-version: '3.7.1'
             tox-env: nodeps-withcov-posix
+            job-name: 'nodeps-test'
+
+          # `noipv6` is created to make sure all is OK on an OS which doesn't
+          # have IPv6 available.
+          # Any supported Python version is OK for this job.
+          # We go with latest micro release for the smalles minor relase
+          # that we support.
+          - python-version: '3.7'
+            noipv6: -noipv6
+            job-name: 'default-tests-noipv6'
+
+          # Just Python 3.8 with default settings.
+          - python-version: '3.8'
+
+          # Just Python 3.9 with default settings.
+          - python-version: '3.9'
+
+          # Windows, minimum Python version with select reactor.
+          - python-version: '3.7'
+            runs-on: 'windows-2022'
+            tox-env: 'alldeps-withcov-windows'
+            job-name: 'win-default-tests-select'
+            # Distributed trial is not yet suported on Windows so we overwrite
+            # the default trial-args to remove concurrent runs and to
+            # select a reactor.
+            trial-args: '--reactor=select'
+
+          # Windows, newest Python supported version with iocp reactor.
+          - python-version: '3.10'
+            runs-on: 'windows-2022'
+            tox-env: 'alldeps-withcov-windows'
+            job-name: 'win-default-tests-iocp'
+            # Distributed trial is not yet suported on Windows.
+            trial-args: '--reactor=iocp'
+
           # On PYPY coverage is very slow (5min vs 30min) as there is no C
           # extension.
           # There is very little PYPY specific code so there is not much to
           # gain from reporting coverage.
           - python-version: pypy-3.7
             tox-env: alldeps-nocov-posix
```

Comment:
```suggestion

          # On PYPY coverage is very slow (5min vs 30min) as there is no C
          # extension.
          # There is very little PYPY specific code so there is not much to
          # gain from reporting coverage.
          - python-version: 'pypy-3.7'
            tox-env: 'alldeps-nocov-posix'
```

Reply:
Ok. This goes to the "delete-lines" and can't be applied by github.
Will manually apply it.